### PR TITLE
Use Xcode 13.4.1 to improve job stability

### PR DESCRIPTION
**Pull request by jmpg93**
There is a bug in [Xcode 13.2](https://developer.apple.com/forums/thread/696504) that is not able to resolve SPM dependencies with the current project configuration (after merging mistica-swiftui inside this repo) and makes de CI job to fail.

As our machines already support Xcode 13.4.1 we can use it for now.

### CODE REVIEW

File: .github/workflows/build-mistica-catalog.yml [hunk 1]
```diff
@@ -45,6 +45,4 @@ jobs:
 
       - name: Upload ipa to AppCenter
         run: npx -p appcenter-cli appcenter distribute release -a Tuenti-Organization/Mistica-iOS -f ./build/ios.ipa -g Public --token ${{ secrets.APPCENTER_API_TOKEN }}
-        
-      - name: Upload dsyms to AppCenter
-        run: npx -p appcenter-cli appcenter crashes upload-symbols -a Tuenti-Organization/Mistica-iOS -s ./build/ios.dSYM --token ${{ secrets.APPCENTER_API_TOKEN }}
```
Comment (jmpg93) [inline .github/workflows/build-mistica-catalog.yml:50]: We can skip this to make builds faster.

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .PHONY: help setup format test simulator archive export clean
 
 # Simulator
-OS_VERSION := 15.2
+OS_VERSION := 15.5
```
Comment (jmpg93) [inline Makefile:4]: The makefile is able to autogenerate the simulator if it doesn't exist.

**Review by amegias — APPROVED** [review]
Do not forget to execute `build-mistica-catalog.yml` before integrating this ;)

**Review by alejandroruizponce — APPROVED** [review]
Great!

File: .github/workflows/build-mistica-catalog.yml (hunk 1 above)
Comment (jmpg93) [reply .github/workflows/build-mistica-catalog.yml:50]: [](url)

> Do not forget to execute `build-mistica-catalog.yml` before integrating this ;)

Deploy working fine https://github.com/Telefonica/mistica-ios/actions/runs/3080655843/jobs/4978259619

**Pull request merged**